src/Mutator/Boolean/LogicalOr.php

```diff
@@ -77,6 +79,72 @@ public function mutate(Node $node): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\BinaryOp\BooleanOr;
+        if (!$node instanceof Node\Expr\BinaryOp\BooleanOr) {
+            return false;
+        }
+
+        if (
+            (
+                $node->left instanceof Node\Expr\BinaryOp\Identical
+                && $node->right instanceof Node\Expr\BinaryOp\Identical
+            ) || (
+                $node->left instanceof Node\Expr\BinaryOp\Equal
+                && $node->right instanceof Node\Expr\BinaryOp\Equal
+            )
+        ) {
+            $varName = '';
+
+            if ($node->left->left instanceof Node\Expr\Variable) {
+                $varName = $node->left->left->name;
```

Comment:
for some reasons, phpstan and phpstorm cannot infer the type Variable to `$node->left->left` although it is checked against the instanceof right before...